app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  before_action :require_admin, only: [:index]
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = current_user
+  end
+
+  # def destroy
+  #   @user = User.find(params[:id])
+  #   @user.destroy
+
+  #   redirect_to users_path, notice: 'User was successfully deleted.'
+  # end
```

Comment:
- [ ] Kindly remove commented code from your project. It makes code hard to read.